solutions/usaco-615.mdx

```diff
@@ -2,16 +2,47 @@
 id: usaco-615
 source: USACO Bronze 2016 February
 title: Milk Pails
-author: Danh Ta Chi Thanh
+author: Ryan Chou, Danh Ta Chi Thanh
 ---
 
 [Official Analysis (Java)](http://www.usaco.org/current/data/sol_pails_bronze_feb16.html)
 
+## Implementation
+
 <LanguageSection>
+	
+<PySection>
 
-<CPPSection>
+```python
+fin = open('pails.in', 'r')
 
-## Implementation
+inputline = list(map(int, fin.readline().split()))
```

Comment:
```suggestion
buck1, buck2, buck3 = map(int, fin.readline().split())
```
also remove 
```
buck1 = inputline[0]
buck2 = inputline[1]
buck3 = inputline[2]
```

Reply:
thanks for the suggestion, i've removed it.